model/components/pinpoint.yml

```diff
@@ -0,0 +1,64 @@
+---
+components:
+
+  # Pinpoint Agent
+  - id: pinpoint-agents
+    name: Pinpoint Agent
+    description: Pinpoint is specialized on distributed system monitoring without changing a single line of code. Provides a solution to help analyze the overall structure of the system and how components within them are interconnected by tracing transactions across distributed applications. Agent is for instrumenting and monitoring Java 6+ applications and gathering host's metrics.
+    link: http://naver.github.io/pinpoint/
+    github:
+      - naver/pinpoint
+    logo: pinpoint
+    license:
+      - Apache License 2.0
+    categories:
+      - agent
+    capabilities:
+      tech:
+        - java
+      aspects:
+        - tracing
+        - metrics [app]
+        - metrics [host]
```

Comment:
Are these really host metrics or do you mean the metrics you get from a JVM?
For example when reading the CPU utilization is the the utilization of the JVM or actually of the host?

If you mean the JVM metrics then change this please to metrics [process].

Reply:
got it:)